config/routes.rb

```diff
@@ -50,6 +50,7 @@
 
     resources :businesses, only: [] do
       resources :conversations, only: :index, controller: :business_conversations
+      resources :invisiblizes, only: :create, module: :businesses
```

Comment:
Why is `module` needed? I don't see it for developers.

Reply:
> Looking good! Outside of tests and some inline change requests this is close to done.
> 
> After that, a few more things:
> 
> 1. I think we dropped the `text` extension for the mailers. Let's make sure to add those back in.
> 2. Let's make sure we have a invisible business seeded.
> 3. Don't forget to add the mailer previews, and check existing ones.

1. Looks like the `text` extension wasn't there before I moved the file so I totally missed that! Good catch, I'll add it.
2. Copy!
3. You got it

Reply:
Because we need to tell the router to look in the business dir not the admin dir.  Developers doesn't have it because that controller is only in the admin namespace

Reply:
> Developers doesn't have it because that controller is only in the admin namespace

I'm confused – shouldn't they both be exactly the same?

Reply:
The developer version is at `app/controllers/admin/invisiblizes_controller.rb` but the business one is at `app/controllers/admin/businesses/invisiblizes_controller.rb` so we need to tell the route to look in the `businesses` directory for the `InvisiblizesController`

Reply:
Got it! Let's move the developer one to match the business one then, too.